### PR TITLE
Update for CN

### PR DESCRIPTION
**Pull request by zlq4863947**

### CODE REVIEW

**Review by nsKpz — CHANGES_REQUESTED** [review]

File: CN/ItemName_CN.txt [hunk 1]
```diff
@@ -33,7 +33,7 @@ ItemName_CN = {
     ItemName_Base.Apple = "苹果",
     ItemName_Base.Apron_Black = "围裙",
     ItemName_Base.Apron_IceCream = "冰淇淋服务员围裙",
-    ItemName_Base.Apron_PileOCrepe = "PileOCrepe 服务员围裙",
+    ItemName_Base.Apron_PileOCrepe = "可丽家 服务员围裙",
```
[the remark this answers is not in the document]
Comment (moonlight-pz) [reply CN/ItemName_CN.txt:36]: Localized translation，for example：（english：Oreo）（chinese：奥利奥）this game's pet（english：Spiffo）(chinese:斯皮福)

File: CN/ItemName_CN.txt [hunk 2]
```diff
@@ -1312,15 +1312,15 @@ ItemName_CN = {
     ItemName_Base.WeldingRods = "焊条",
     ItemName_Base.WestpointMap = "地图（西点镇）",
     ItemName_Base.WhiskeyEmpty = "瓶子（空）",
-    ItemName_Base.WhiskeyFull = "威士忌瓶",
+    ItemName_Base.WhiskeyFull = "威士忌",
```
[the remark this answers is not in the document]
Comment (zlq4863947) [reply CN/ItemName_CN.txt]: modified to 波本威士忌

File: CN/ItemName_CN.txt [hunk 3]
```diff
@@ -1127,15 +1127,15 @@ ItemName_CN = {
...
+    ItemName_Base.SunflowerSeeds = "葵花瓜子",
     ItemName_Base.SutureNeedle = "缝合针",
     ItemName_Base.SutureNeedleHolder = "缝合钳",
     ItemName_Base.SwimTrunks_Blue = "泳裤",
     ItemName_Base.SwimTrunks_Green = "泳裤",
     ItemName_Base.SwimTrunks_Red = "泳裤",
     ItemName_Base.SwimTrunks_Yellow = "泳裤",
     ItemName_Base.Swimsuit_TINT = "泳衣",
-    ItemName_Base.TVDinner = "微波便当",
+    ItemName_Base.TVDinner = "冷冻便当",
```
[the remark this answers is not in the document]
Comment (zlq4863947) [reply CN/ItemName_CN.txt]: After our discussion, it has been changed to: `快捷盒饭`。
By the way, there is no non-expiring bento in the real world

**Comment by moonlight-pz** [comment]
> @zlq4863947 @moonlight-pz @Mishasama
> 
> Hey guys.
> Any further updates to the chinese translation? Last update has been a while :)

I have submitted：https://github.com/project-zomboid-cn-translation/ProjectZomboidTranslations/pull/26
Will be submitted to you after review within the group

**Comment by Mishasama** [comment]
> @zlq4863947 @moonlight-pz @Mishasama
> 
> Hey guys.
> Any further updates to the chinese translation? Last update has been a while :)

Sorry, I will not push any more updates for CHS since they take control dominance. I can't guarantee the quality of the translation under that.

**Pull request merged**